scanner/e2etests/testdata/image_tests.json

```diff
@@ -5201,6 +5202,7 @@
         "required_feature_flag": null
     },
     {
+        "disabled_reason": "ROX-20730: CVE-2022-22978 not found",
```

Comment:
curious do we have Jiras or plans to address these before we release 4.4?

Reply:
There is a JIRA, right? ROX-20730?